community/Wings Fail2ban Setup.md

```diff
@@ -0,0 +1,66 @@
+# Fail2ban wings configuration
+
+Fail2ban is a program that works with iptables to better secure your server and services from brute force login attempts
+
+### Step 1: if you havent already Using your preferred package manager
+(to my best knowledge, iptables will come with fail2ban when you install it, but if not install that too)
```

Comment:
Try to keep it simple here: "Step 1: Install fail2ban", most people wont try to install it if they already have it.